CHANGELOG.markdown

```diff
@@ -1,7 +1,8 @@
-next [????.??.??]
+4.19 [????.??.??]
 -----------------
 * Add `{Functor,Foldable,Traversable}WithIndex` instances for
   `Control.Applicative.Const` and `Data.Functor.Constant.Constant`.
+* Make `<>~` and `<>=` require only `Semigroup`, not `Monoid`
```

Comment:
Don't forget to mention the other functions that were changed!

Reply:
Done